SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -616,6 +645,169 @@ def __init__(self, master=None):
                         idx += 1
                         self.load_cfg_file(sub_path, idx, False)
 
+    def search_widget_label(self):
+        self.current_matches = []
+        self.current_match_index = -1
+
+        def expand_tree_to_page(page_id_list):
+            level_3_list = []
+            for level_1 in self.left.get_children():
+                for level_2 in self.left.get_children(level_1):
+                    for level_3 in self.left.get_children(level_2):
+                        level_3_list.append(level_3)
+            print(f'level3: {level_3_list}')

```

Comment:
These prints appear to be left over from debugging, correct? If so, they can be removed.